ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -126,20 +124,38 @@ def prepare(self):
         """
 
     def next_record(self):
-        for key in ENTITY_TYPES["Table"]:
-            self.service = self.get_database_service()
-            self.tables[key] = self.atlas_client.list_entities(entity_type=key)
+        for service in self.service_connection.databaseServiceName or []:
```

Comment:
Is this a list of Atlas service connections?

Reply:
So , this is list of `service name` user will provide to get metadata for those service